### PR TITLE
Update rm2fb

### PR DESCRIPTION
**Pull request by Eeems**
Only restart xochitl if it's running.
Reset crashnum before restarting xochtil.
Modify preremove message based on xochitl's state.

### CODE REVIEW

File: package/rm2fb/package [hunk 1]
```diff
@@ -46,11 +46,20 @@ package() {
 configure() {
     systemctl daemon-reload
     systemctl enable rm2fb --now
-    systemctl restart xochitl
+    # Restart xochitl if it's running
+    if systemctl --quiet is-active xochitl; then
```
Comment (raisjn) [inline package/rm2fb/package:50]: the exit code is 0 if xochitl is active, so should this be a negated condition?

**Review by raisjn — CHANGES_REQUESTED** [review]
nice usage of systemctl!

i think need to bump version of rm2fb, too

File: package/rm2fb/package (hunk 1 above)
Comment (Eeems) [reply package/rm2fb/package:50]: Oh right, forgot about that.

Comment (Eeems) [reply package/rm2fb/package:50]: Hmm, after a quick test, this actually works properly and will be truthy if systemctl is running.

**Review by raisjn — DISMISSED** [review]

**Review by raisjn — APPROVED** [review]

**Pull request merged**